### DISCUSSION TITLE
Can't access the czlendar

### DISCUSSION THREAD

**Discussion by DKRDev** [spatie/laravel-google-calendar#217] [Q&A]
Getting this error code

{ "error": { "errors": [ { "domain": "global", "reason": "notFound", "message": "Not Found" } ], "code": 404, "message": "Not Found" } }

<img width="1432" alt="Capture d’écran 2021-07-11 à 00 39 08" src="https://user-images.githubusercontent.com/53798944/125179716-72360980-e1e0-11eb-979d-21e4f1dca047.png">

**Comment by pgumpoldsberger** [comment]
I had the same error and was pulling my hair out debugging it.

In the end it was as simple as adding the mail of the user which I would like to impersonate in the .env file. This was necessary because I delegated domain-wide access.  

I added the following line to my .env-file:
`GOOGLE_CALENDAR_IMPERSONATE="user@domain.com"`
